migrations/migrations/20181101224717-create-user.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      name: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      lastName: {
```

Comment:
If you use `underscored: true`, attributes should be in snake_case like `last_name` (only in migration)